### DISCUSSION TITLE
EJBCA CE 7.9.0.1

### DISCUSSION THREAD

**Discussion by hesunmark** [Keyfactor/ejbca-ce#41] [Announcements]
Release highlights include: 

- More frequent releases – Community releases will follow the EJBCA Enterprise release schedule including all major and feature releases.  
- More flexible certificate enrollment via REST API – a subset of the EJBCA REST API is now supported, allowing for certificate enrollment and revocation.  
- External Account Bindings – certificates can be associated with identities from external data sources. 

For more information  

Read the full [EJBCA 7.9.0.1 Release Notes](https://doc.primekey.com/x/HgYVAw) 

Visit the EJBCA project on [ejbca.org](https://www.ejbca.org)

<hr /><em>This discussion was created from the release <a href='https://github.com/Keyfactor/ejbca-ce/releases/tag/EJBCA_7_9_0_1_20220525'>EJBCA CE 7.9.0.1</a>.</em>

**Comment by svenska-primekey** [comment]
Great update to share! Time to update the Ansible playbook to get EJBCA CE from GitHub now.

**Comment by biau83** [comment]
<img width="687" alt="EJBCA-CE-7 9" src="https://user-images.githubusercontent.com/8871883/173766119-33132fe2-cddd-4ba0-b09d-ee71df64446a.png">
Hi guys, it seem like REST is all disabled ;)

**Reply by celolam** [reply]
Hello friend!! Same here.
If you find any solution or any new about how to fix this, please tell me, I would really appreciate it :)

**Comment by bergstiefelbene** [comment]
Internal

Hi,

thank you for your fats response! What should be activated exactly?
***@***.***

Mit freundlichen Grüßen/Kind regards
Bernhard Schmid

IT WIO Sec&DCO Directory Services/IT Security & DC Operations (Sec&DCO) (FIIO4)

ZF Group
Corporate Finance, IT, M&A
ZF Friedrichshafen AG
Ehlersstraße 50, 88046 Friedrichshafen, Deutschland/Germany
Telefon/Phone: +49 7541 77-969812
***@***.******@***.***>

Vorsitzender des Aufsichtsrats/Chairman of the Supervisory Board: Dr. Heinrich Hiesinger
Vorstand/Board of Management: Wolf-Henning Scheider (Vorsitzender/CEO), Dr. Konstantin Sauer, Dr. Martin Fischer, Sabine Jaskula, Dr. Holger Klein, Wilhelm Rehm, Stephan von Schuckmann
Sitz/Headquarters: Friedrichshafen
Handelsregistereintrag Amtsgericht Ulm HRB 630206/Trade register of the municipal court of Ulm HRB 630206

Informationen zur Verarbeitung Ihrer Daten und zu Ihren Rechten erhalten Sie jederzeit in unseren Datenschutzhinweisen: https://www.zf.com/de/data-protection-notice
You can find information about how we process your data and your rights in our data protection notice: https://www.zf.com/en/data-protection-notice
­
Von: biau83 ***@***.***>
Gesendet: Mittwoch, 15. Juni 2022 09:16
An: Keyfactor/ejbca-ce ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: Re: [Keyfactor/ejbca-ce] EJBCA CE 7.9.0.1 (Discussion #41)


[EJBCA-CE-7 9]<https://user-images.githubusercontent.com/8871883/173766119-33132fe2-cddd-4ba0-b09d-ee71df64446a.png>
Hi guys, it seem like REST is all disabled ;)

-

**Comment by biau83** [comment]
it is a fresh installation, without doing anything, REST by default is disabled. And i don't see anyway to enable it. 
(but i am perfectly fine without REST in CE, so that we can push ppl for EE)  >:D

**Reply by hesunmark** [reply]
This issue has now been fixed. Please see the tagged version EJBCA CE 7.9.0.2

**Comment by primetomas** [comment]
Turns out this is actually a bug.
https://jira.primekey.se/browse/ECA-10798 (not reachable externally right now due to an upgrade)
It will be fixed ASAP and come out as a 7.9.0.2.
Thanks for the report!
